modules/cnf-about-topology-aware-lifecycle-manager-policies.adoc

```diff
@@ -18,4 +18,4 @@ For policies that update an Operator subscription with manual approval, {cgu-ope
 
 For more information about managed policies, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/governance/index#policy-overview[Policy Overview] in the {rh-rhacm} documentation.
 
-For more information about the `PolicyGenTemplate` CRD, see the "About the PolicyGenTemplate" section in "Deploying distributed units at scale in a disconnected environment".
+For more information about the `PolicyGenTemplate` CRD, see the "About the PolicyGenTemplate CRD" section in "Configuring managed clusters with policies and PolicyGenTemplate resources".
```

Comment:
Should we link to the appropriate section here so that the reader does not have to manually search for it (like the section above)?